addons/web/static/src/js/widgets/date_picker.js

```diff
@@ -24,7 +24,8 @@ var DateWidget = Widget.extend({
         this._super.apply(this, arguments);
 
         this.name = parent.name;
-        this.options = _.extend({
+        var locale = options.defaultDate._locale ? {locale: options.defaultDate._locale._abbr} : {};
```

Comment:
This should probably be:
`var locale = options.defaultDate ? {locale: options.defaultDate._locale._abbr} : {};`

Reply:
On the runbot `option.defaultDate` was defined but not `option.defaultDate._locale`. But let me change that, I'm not sure defaultDate is defined every time